codegen/src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -310,10 +312,9 @@ private static MethodSpec buildConstructor(Class authType, String authName) {
                 .addParameter(String.class, CONTRACT_ADDRESS)
                 .addParameter(Web3j.class, WEB3J)
                 .addParameter(authType, authName)
-                .addParameter(BigInteger.class, GAS_PRICE)
```

Comment:
We don't want to force people to use GasProviders in generated wrappers until we release version 4.0. I.e. generatd code should support both gas providers and the original wrappers (which are correctly marked as deprecated).

Reply:
ofcourse